Description: As /usr/bin/lppasswd is not anymore shipped with the setuid
 bit set, document how one can use dpkg-statoverride to ensure the
 change stays across upgrades.
Origin: vendor
Author: Didier Raboud <odyx@debian.org>
Last-Update: 2014-03-11
--- a/man/lppasswd.man
+++ b/man/lppasswd.man
@@ -50,7 +50,16 @@
 must be made setuid to root with the command:
 .br
 .nf
+.RS
 chmod u+s lppasswd
+.RE
+.fi
+This change will be overridden by further updates of the cups\-client package.
+The following command ensures that it will be preserved:
+.nf
+.RS
+dpkg-statoverride \-\-add root root 4755 /usr/bin/lppasswd
+.RE
 .fi
 .PP
 While every attempt has been made to make \fIlppasswd\fR secure against
